README.rst

```diff
@@ -32,12 +32,12 @@ Install
 Pre-Install
 -----------
 
-Linux (Debian)
+Linux (Debian/Ubuntu)
 ^^^^^^^^^^^^^^
```

Comment:
Please extend the underline to match the section title - otherwise, `docutils` and Sphinx will complain.